src/Analytics/UAExperimentData.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace PhpAb\Analytics;
+
+use PhpAb\Test;
+
+/**
+ * This is just a concept, no code review required
+ */
+class UAExperimentData
```

Comment:
Yep!!
Better name would be ...DataCollector. But that's the concept.
This class could simply listen to the events and fills itself with data. (see Event\SubscriberInterface)